Python/Product/PythonTools/PythonToolsPackage.cs

```diff
@@ -469,6 +469,9 @@ protected override object GetAutomationObject(string name)
             // test discoverer and test executor to connect back to VS.
             Environment.SetEnvironmentVariable("_PTVS_PID", Process.GetCurrentProcess().Id.ToString());
 
+            // Setup the debug watcher so we can query for IDebug types
+            var x = DebuggerHelper.Instance;
```

Comment:
This can use `_` to indicate that the returned value is ignored.

But, better yet, since it relies on side effects from the getter, I think `Instance` should become `GetInstance()` instead.